libraries/core/src/main/kotlin/com/bumble/appyx/core/navigation/NoOpNavModel.kt

```diff
@@ -0,0 +1,17 @@
+package com.bumble.appyx.core.navigation
+
+import com.bumble.appyx.core.navigation.onscreen.OnScreenStateResolver
+
+/**
+ * An implementation of a NavModel that won't add any children.
+ * This is potentially useful if your ParentNode only uses
+ * [com.bumble.appyx.core.navigation.model.permanent.PermanentNavModel]
+ */
+class NoOpNavModel<NavTarget, State> : BaseNavModel<NavTarget, State>(
```

Comment:
Would prefer `EmptyNavModel`. I think will be easier to find naturally via autocomplete.

Reply:
done